lib/collection/src/operations/types.rs

```diff
@@ -229,7 +229,7 @@ impl From<CollectionConfigInternal> for CollectionConfig {
             optimizer_config,
             wal_config: Some(wal_config),
             quantization_config,
-            strict_mode_config,
+            strict_mode_config: strict_mode_config.map(StrictModeConfigOutput::from),
```

Comment:
`CollectionConfigInternal` is used in collection telemetry, and, therefore, should either be splitted into Output structure, or contain `StrictModeConfigOutput`